apps/projects/app/components/Content/IssueDetail/components/Detail.js

```diff
@@ -0,0 +1,247 @@
+      </Text.Block>
+      <Text.Block>{date}</Text.Block>
+    </div>
+  </Wrapper>
+)
+
+const fakeActivities = [
+  {
+    name: 'Worf',
+    log: 'began the task',
+    date: '2 days ago',
+    avatar: null,
+  },
+  {
+    name: 'Tasha Yar',
+    log: 'assigned Worf',
+    date: '3 days ago',
+    avatar: null,
+  },
+  {
+    name: 'Data',
+    log: 'rejected Jean-Luc\'s work',
+    date: 'Last seen 4 hours ago',
+    avatar: null,
+  },
+]
+
+const fakeMembers = [
+  {
+    name: 'Worf',
+    role: 'Contributor',
+    status: 'Pending assignment',
+    avatar: null,
+  },
+  {
+    name: 'Tasha Yar',
+    role: 'Contributor',
+    status: 'Assignment approved',
+    avatar: null,
+  },
+  {
+    name: 'Data',
+    role: 'Task Manager',
+    status: 'Last seen 4 hours ago',
+    avatar: null,
+  },
+]
+
+const Detail = ({
+  bountyBadge = '100 ANT',
+  title,
+  number,
+  repo,
+  body,
+  activities = fakeActivities, // TODO: Remove default fake value when data arrives from backend
+  team = fakeMembers, // TODO: Also this
+  summaryData, // an object containing {experience, deadline, slots, status} for that issue
+}) => {
+  const calculatedDate = () => {
+    // something obtained from props and transformed to be like:
+    return 'Opened 4 days ago'
+  }
+  // Some dynamically generated components
+  const teamRows = team.map((member, i) => <MemberRow key={i} {...member} />)
+  const activityRows = activities.map((data, i) => (
+    <ActivityRow key={i} {...data} />
+  ))
+  return (
+    <Wrapper>
+      <div style={{ flex: 3, maxWidth: '705px' }}>
+        <div style={cardStyle}>
+          <Wrapper style={{ justifyContent: 'space-between' }}>
+            <div style={{ ...column, flex: 2, marginRight: '20px' }}>
+              <Text.Block size="xlarge" style={{ marginBottom: '10px' }}>
+                {title}
+              </Text.Block>
+              <Text.Block color="#21AAE7" style={{ marginBottom: '10px' }}>
+                <GithubLogo src={githubLogo} />
+                {repo} #{number}
+              </Text.Block>
+              <Text.Block
+                size="small"
+                color={theme.textSecondary}
+                style={{ marginBottom: '10px' }}
+              >
+                {calculatedDate()}
+              </Text.Block>
+            </div>
+            <div style={{ ...column, flex: 0, alignItems: 'flex-end' }}>
+              <DropDownButton enabled />
+              {/* <Text>$500 USD</Text> */}
+              <Badge
+                foreground={theme.badgeNotificationBackground}
+                background="#D0F2DB"
+                style={{ marginTop: '15px' }}
+              >
+                <Text>{bountyBadge}</Text>
+              </Badge>
+            </div>
+          </Wrapper>
+          <SummaryTable {...summaryData} />
+          <FieldTitle>Description</FieldTitle>
+          <Text.Block style={{ marginTop: '20px' }}>{body}</Text.Block>
+          <Text.Block>Labels</Text.Block>
```

Comment:
Labels are present in issue - should they be displayed here?

Reply:
Solved at 871d29ba235154802b0e0ffa7b702c7264ce29d0